pkg/policy/selectorcache.go

```diff
@@ -294,6 +331,12 @@ func (s *selectorManager) setSelections(selections *[]identity.NumericIdentity)
 type fqdnSelector struct {
 	selectorManager
 	selector api.FQDNSelector
+	user     identityPopulator
```

Comment:
IMO `user` here is misleading w.r.t. `selectorManager.users`. Maybe `fqdnManager` instead?